poradnia/users/locale/en/LC_MESSAGES/django.po

```diff
@@ -98,10 +98,14 @@ msgstr ""
 msgid "Keys"
 msgstr ""
 
-#: templates/users/user_detail.html:41 templates/users/user_filter.html:43
+#: templates/users/user_filter.html:43
 msgid "Client cases"
 msgstr ""
 
+#: templates/users/user_detail.html:41
+msgid "My cases"
+msgstr ""
```

Comment:
Dlaczego brak tłumaczenia?

Reply:
Tam są dwa package - en i pl, w każdym osobne LC_MESSAGES - w packagu pl są tłumaczenia, tam zmieniłam, a w en są właśnie puste stringi wszędzie - dodałam dla spójności, czyli brak tłumaczenia w wersji angielskiej, ale jestem ciekawa dlaczego tak - czy tego wymagda moduł trans w django?

Reply:
Masz rację, to język angielski, czego nie zauważyłem.